_im-plugin/ism/policies.md

```diff
@@ -438,6 +439,103 @@ Parameter | Description | Type | Required
+                "name": "rollup",
+                "actions": [
+                    {
+                        "rollup": {
+                            "ism_rollup": {
+                                "description": "Creating rollup through ISM",
+                                "target_index": "target",
+                                "page_size": 1000,
+                                "dimensions": [
+                                    {
+                                        "date_histogram": {
+                                            "fixed_interval": "60m",
+                                            "source_field": "order_date",
+                                            "target_field": "order_date",
+                                            "timezone": "America/Los_Angeles"
+                                        }
+                                    },
+                                    {
+                                        "terms": {
+                                            "source_field": "customer_gender",
+                                            "target_field": "customer_gender"
+                                        }
+                                    },
+                                    {
+                                        "terms": {
+                                            "source_field": "day_of_week",
+                                            "target_field": "day_of_week"
+                                        }
+                                    }
+                                ],
+                                "metrics": [
+                                    {
+                                        "source_field": "taxless_total_price",
+                                        "metrics": [
+                                            {
+                                                "sum": {}
+                                            }
+                                        ]
+                                    },
+                                    {
+                                        "source_field": "total_quantity",
+                                        "metrics": [
+                                            {
+                                                "avg": {}
+                                            },
+                                            {
+                                                "max": {}
+                                            }
+                                        ]
+                                    }
+                                ]
+                            }
+                        }
+                    }
+                ],
+                "transitions": []
+            }
+        ]
+    }
+}
+````
+
+#### Field parameters
+
+The following field parameters are required:
+
+Field Parameter | Data Type | Description 
+:--- | :--- |:---
+source_field | String | The field(s) to transform.
```

Comment:
Does this accept a comma-separated list or a wildcard pattern? If so, we should state that.

Reply:
This was a mistake on my part. I'll re-write it to be singular.